index.html

```diff
@@ -384,9 +384,9 @@ <h2 style="color:white" class="section-heading">Contact Us</h2>
                     <span class="error" id="error_message"></span>
                   </div>
                   <div class="form_group">
-                    <div class="row">
+                    <div class="row" style="margin: 0 auto; width: 656px; text-align: center;" >
                       <span id="success" class="col-xs-4"></span>
```

Comment:
This is what I missed in the first review, This being a row out of which 4 columns are occupied by this, center block alone doesn't seem to be working. And, without center block we get the deviation in different screens that we were looking at.